<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_S">
     <property name="title">
      <string>编码(&amp;S)</string>
     </property>
     <addaction name="action_GB2312"/>
     <addaction name="action_UTF_8"/>
    </widget>
    <addaction name="action_Redo"/>
    <addaction name="action_Undo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Past"/>
    <addaction name="separator"/>
    <addaction name="menu_S"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="action_Close"/>
    <addaction name="action_CloseAll"/>
    <addaction name="separator"/>
    <addaction name="action_Tile"/>
    <addaction name="action_Cascade"/>
    <addaction name="separator"/>
    <addaction name="action_Next"/>
    <addaction name="action_Previous"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="action_SaveAs"/>
   <addaction name="separator"/>
   <addaction name="action_Exit"/>
   <addaction name="separator"/>
   <addaction name="action_Redo"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Past"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/新建文件.png</normaloff>:/new/prefix1/img/新建文件.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/打开文件.png</normaloff>:/new/prefix1/img/打开文件.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/保存.png</normaloff>:/new/prefix1/img/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/另存为.png</normaloff>:/new/prefix1/img/另存为.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/退出.png</normaloff>:/new/prefix1/img/退出.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;R)</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/退出.png</normaloff>:/new/prefix1/img/退出.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;R)</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/退出.png</normaloff>:/new/prefix1/img/退出.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;U)</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/帮助.png</normaloff>:/new/prefix1/img/帮助.png</iconset>
   </property>
   <property name="text">
    <string>剪贴(&amp;T)</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/新建文件.png</normaloff>:/new/prefix1/img/新建文件.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_Past">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/打开文件.png</normaloff>:/new/prefix1/img/打开文件.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="action_GB2312">
   <property name="text">
    <string>GB2312</string>
   </property>
  </action>
  <action name="action_UTF_8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/帮助.png</normaloff>:/new/prefix1/img/帮助.png</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;O)</string>
   </property>
  </action>
  <action name="action_CloseAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/打开文件.png</normaloff>:/new/prefix1/img/打开文件.png</iconset>
   </property>
   <property name="text">
    <string>关闭所有(&amp;A)</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/打开文件.png</normaloff>:/new/prefix1/img/打开文件.png</iconset>
   </property>
   <property name="text">
    <string>平铺(&amp;T)</string>
   </property>
  </action>
  <action name="action_Cascade">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/文件.png</normaloff>:/new/prefix1/img/文件.png</iconset>
   </property>
   <property name="text">
    <string>层叠(&amp;C)</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/窗口.png</normaloff>:/new/prefix1/img/窗口.png</iconset>
   </property>
   <property name="text">
    <string>下一个(&amp;N)</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/新建文件.png</normaloff>:/new/prefix1/img/新建文件.png</iconset>
   </property>
   <property name="text">
    <string>上一个(&amp;P)</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/帮助.png</normaloff>:/new/prefix1/img/帮助.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/img/帮助.png</normaloff>:/new/prefix1/img/帮助.png</iconset>
   </property>
   <property name="text">
    <string>关于QT(&amp;O)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
